### ISSUE TITLE
Question: Multiple sensors from a single query

### ISSUE THREAD

**Issue by jounathaen** [pierosavi/pierosavi-imageit-panel#122]
Hi,

First of all: Thanks for providing this plugin. I have a question on the usage of the plugin and despite this being out-of-maintenance, maybe somebody can help me:

### What I achieved

I have an image with 20 sensors. The sensor values come are stored in InfluxDB. What works for me, is creating 20 different queries and then referring to them as `Query ID`.

So my 20 Queries look like that:

Sensor 1a:
| time | _value |
|---|---|
|1.2.22... | 42 |

Sensor 2a:
| time | _value |
|---|---|
|1.2.22... | 21 |

 ...

### Desired situation

The 20 queries are rather slow. I think it would be way better to use a single query to build a table with all sensors, and  then just take the value from there. But I don't know how to do that. Honestly, I didn't even understand what is meant by `Query Alias`.

query1:

| time | sensor1 | sensor 2 |
|---|---|---|
| 1.2.22... | 42 | 21 |

**TLDR: How to add 2 sensors with the values from above's table coming from a single query?**

**Comment by peishaofeng** [comment]
use "group by" will generate multiple series from one query clause

e.g.

```
SELECT last("value") FROM "air.temperature" WHERE ("factory" = 'F3') AND $timeFilter GROUP BY "location"
```

**Comment by jounathaen** [comment]
Thanks, but how to use this in imageit then?

**Comment by peishaofeng** [comment]
A rendered example with data from InfluxDB

![imageit-air-temperature](https://user-images.githubusercontent.com/16284900/169188865-7c1c6449-2843-4693-a173-eec2357c40bb.png)

2 queries to generate multiple time-series data
![query-alias](https://user-images.githubusercontent.com/16284900/169189124-62891999-c2f6-4ae9-8f3a-306a3178fcba.png)

`GROUP BY "location"` will generate multiple series aliased by its `location` label e.g. `S2-1-T`, `S3-1-T` from `$tag_location-T`, `S2-1` & `S3-1` denote series alias grouped by  `location`

refer to those aliased series in `Query Alias` as below sensor

![sensor-by-query-alias](https://user-images.githubusercontent.com/16284900/169189952-3f98d8de-ee54-41aa-ae1a-5f889efb44c2.png)

Hope can help you.

**Comment by jounathaen** [comment]
Thanks a lot for the helpful answer! I'll try this!
